versioned_plugins.rb

```diff
@@ -30,8 +30,7 @@ class VersionedPluginDocs < Clamp::Command
     "logstash-input-java_input_example",
     "logstash-filter-java_filter_example",
     "logstash-output-java_output_example",
-    "logstash-codec-java_codec_example",
-    "logstash-integration-logstash"
```

Comment:
Please consider renaming the PR to "Remove integration-logstash from skiplist" because I search on that a lot.

Reply:
Yup, I have changed.